extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1870,4 +1873,30 @@ export class AmpStoryPage extends AMP.BaseElement {
   isAutoAdvance() {
     return this.advancement_.isAutoAdvance();
   }
+
+  /**
+   * Set the data-orig-tabindex to the default tabindex of tabbable elements
+   */
+  initializeTabbableElements_() {
+    scopedQuerySelectorAll(this.element, Selectors.ALL_TABBABLE).forEach(
+      (el) => {
+        el.setAttribute('data-orig-tabindex', el.getAttribute('tabindex') || 0);
```

Comment:
Is a benefit to storing this on the element rather saving a reference in JS like the pattern in modal.js?
https://github.com/ampproject/amphtml/blob/master/src/modal.js#L48

Reply:
They sound very similar, I'd say that having an extra structure to store these values as in `modal.js` would be overly complicated (modal.js stores a lot more in that stack), maybe `el.origTabindex = ...` as a property would be a better way to store the value than the stack.

All considered, if someone sets the `tabindex` on a link on the source doc, it would be weird to replace it with `-1` on inactive pages without showing the original value on the dom somehow, that's why I think it makes sense to keep it as an attribute. Do you see any benefits of setting it as a JS property instead?